content/pricing/billing.textile

```diff
@@ -33,3 +33,17 @@ To update your billing details:
 You can update your payment details and view your invoices in this screen.
 
 It's also possible to have your invoices sent to an accounts department by filling in the *Optional billing email address*.
+
+h2(#alerts). Billing alerts
+
+Billing alerts notify you by email when your monthly spending exceeds a set amount on the accounts where you have paid "packages":/docs/pricing#packages (standard or pro). This feature helps you monitor costs and avoid unexpected charges.
+
+The following steps guide you through setting up billing alerts:
+
+* Log in to your Ably "account.":https://ably.com/login
+* Go to the *Account* menu in the top navigation bar.
+* Select *Billing*.
+* Scroll down to the *Billing alerts* section.
+* In the *Create new alert* section, enter the email address where you want to receive the alert. You can select from the dropdown if an address is already listed.
```

Comment:
They can't enter any additional email addresses but have to select from the dropdown. The choices will be

**As Owner**
* Themselves
* Billing users

**As a billing user**
* Themselves